src/features/workflow/WorkflowPage.tsx

```diff
@@ -19,7 +20,7 @@ interface WorkflowPageProps {
 const WorkflowPage: React.FC<WorkflowPageProps> = ({ qriRef }) => {
   const dispatch = useDispatch()
   let dataset = useSelector(selectWorkflowDataset)
-  const latestRun = useSelector(selectLatestDryRun)
+  const latestRun = useSelector(selectRun(useSelector(selectLatestDryRunId)))
```

Comment:
separate into two lines, one selecting `latestRunID` and the other selecting `latestRun`